conf/conda_local.config

```diff
@@ -8,6 +8,12 @@ conda {
 
 process {
 
+    errorStrategy = { task.attempt < 3 ? 'retry' : 'ignore' }
+
+    time = '1h'
```

Comment:
I suggest that we leave the time limit to `3h` here since systems with `conda` would generally not have high cpu/mem